pytest_rts/utils/git.py

```diff
@@ -1,26 +1,38 @@
 """This module contains code for git operations"""
 import re
 import subprocess
-from typing import Dict, List, Set
+from typing import List, Set
 
 from pydriller import GitRepository
 
 
+def commit_exists(commithash: str) -> bool:
+    """Check if a given commithash exists in the Git repository"""
+    return commithash in [commit.hash for commit in get_git_repo().get_list_commits()]
+
+
 def get_changed_files_current(repo: GitRepository) -> List[str]:
     """Get changed files in git working directory"""
     return repo.repo.git.diff("--name-only").split()
 
 
-def get_file_diff_data_current(repo: GitRepository, filename: str) -> str:
+def get_changed_files_between_commits(
+    repo: GitRepository, commithash1: str, commithash2: str
+) -> List[str]:
+    """Get changed files between two commits"""
+    return repo.repo.git.diff("--name-only", commithash1, commithash2).split()
```

Comment:
If I'm not mistaken, in git diff command last commit hash could be omitted and that will automatically do diff between the first commit hash and HEAD. Please check.

Reply:
Giving only the first commithash results in both workdir and committed changes to be displayed in the diff so I used that.